Consequences/Results/Result.cs

```diff
@@ -8,6 +8,11 @@ public Result(ResultItem[] resultItems)
     _resultItems = resultItems;
   }
 
+  public ResultItem[] GetResultItems()
```

Comment:
a nice thing about unit tests is that you implement an activity that seeks to reach a goal (like writing results) and you find that your interfaces or implementations are missing things. That helps you ensure your code does what it needs to do. Great find and fix.